### DISCUSSION TITLE
tutorials/assets_pipeline/importing_images

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#150] [User-contributed notes]
# tutorials/assets_pipeline/importing_images

Supported image formats: Godot can import the following image formats: BMP (.bmp) - No support for 16-bit per pixel images. Only 1-bit, 4-bit, 8-bit, 24-bit, and 32-bit per pixel images are support...

https://docs.godotengine.org/en/4.3/tutorials/assets_pipeline/importing_images.html

**Comment by Egalexandr** [comment]
You can use GIMP to export image of desired format using '**File/Export As**' then '**select file type**' - **DDS Image**.
You may use compression as **None**, there nothing to say...
Or set it **BC1/DXT1** for images with none alpha (most compressed, almost _unnoticeable_ artifacts)
Or set it **BC3/DXT5** for images with alpha channel (size*2 in comparison of BC1/DXT1)

Imported image will be of type **ImageTexture** (.dds format), and **no** *.import created
